commit 8071ee0bf2f1dac1974b5907c3a61a205bac8dae (refs/changes/59/2259/2)
Author: Jared Morrow <jm@joyent.com>
Date:   2017-07-24T21:15:53+00:00 (2 years, 2 months ago)
    
    MANTA-3109 manta log uploader retries multiple times in parallel
    Reviewed by: Joshua M. Clulow <jmc@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/deps/manta-scripts b/deps/manta-scripts
index cd72e80..4b86dd1 160000
--- a/deps/manta-scripts
+++ b/deps/manta-scripts
@@ -1 +1 @@
-Subproject commit cd72e801576de5a4b1aec35a7f79409d3f1eebb5
+Subproject commit 4b86dd132d68cb177f066a83d90b8fdc9ac2cde9
